apps/src/componentLibrary/tabs/_Tab.tsx

```diff
@@ -97,19 +97,26 @@ const _Tab: React.FunctionComponent<TabsProps> = ({
   isClosable = false,
   onClose = () => {},
 }) => {
+  const [longTooltip, setLongTooltip] = useState<TooltipProps>();
   const handleClick = useCallback(() => onClick(value), [onClick, value]);
   const handleClose = useCallback(() => onClose(value), [onClose, value]);
-  const handleNativeTooltip = (e: React.MouseEvent<HTMLButtonElement>) => {
+  const handleNativeTooltip = (
+    e: React.MouseEvent<HTMLButtonElement> | React.FocusEvent<HTMLButtonElement>
+  ) => {
     const target = e.currentTarget;
     // Locate the element that contains the text (if nested inside spans or other elements).
     const textElement = target.querySelector('span') || target;
 
-    if (textElement.scrollWidth > textElement.clientWidth && !tooltip) {
-      // Set the tooltip text if overflow occurs if the tooltip prop is not passed.
-      target.title = textElement.textContent || '';
+    if (textElement.scrollWidth > textElement.clientWidth) {
+      // Not sure what the ID here should be.
+      setLongTooltip({
+        tooltipId: 'test-id',
+        text: text || '',
+        direction: 'onBottom',
+      });
     } else {
-      // Clear tooltip if no overflow.
-      target.title = '';
+      // is unset necessary / should be somewhere else (onBlur)?
```

Comment:
hide/show dsco tooltip is covered in the dsco `Tooltip` component itself.
What you have here should be enough (unless we decided to add target.title again)